pom.xml

```diff
@@ -3,8 +3,6 @@
     <modules>
         <module>javaparser-core</module>
         <module>javaparser-testing</module>
-        <module>javaparser-core-generators</module>
```

Comment:
Ok, we do not want to distribute the generators but will IDE consider this modules if we remove them from the POM?